settings/settings.go

```diff
@@ -72,12 +72,19 @@ type Disks struct {
 }
 
 type DiskSettings struct {
-	ID             string
-	DeviceID       string
-	VolumeID       string
-	Lun            string
-	HostDeviceID   string
-	Path           string
+	ID           string
+	DeviceID     string
+	VolumeID     string
+	Lun          string
+	HostDeviceID string
+	Path         string
+
+	// iscsi related
```

Comment:
let's also namespace all iscsi settings under "iscsi" (ISCSIDiskSettigns struct) to avoid any future conflicts and ambiguity.